### ISSUE TITLE
AsyncAPI open meeting, Tuesday Nov 24 2020 8AM UTC

### ISSUE THREAD

**Issue by derberg** [asyncapi/asyncapi#462]
This is the meeting for the AsyncAPI Special Interest Group (SIG). You're invited to [join us](https://zoom.us/j/165106914) and ask questions. **The meeting takes place on alternate Tuesdays**. Recordings from the previous meetings are available in [this](https://www.youtube.com/watch?v=S8gvf0XjO10&list=PLbi1gRlP7pijUwZJErzyYf_Rc-PWu4lXS) playlist on YouTube.

**This time we meet at [8AM UTC](https://everytimezone.com/s/2088528d)**

If you want to make proposals for the specification and/or the tooling, please reach out in our [Slack workspace](https://www.asyncapi.com/slack-invite) or just leave a comment below.

Join [this](https://groups.google.com/forum/#!forum/asyncapi-users) mailing list to get an always-up-to-date invite to the meeting in your calendar. 

## Attendees

> Join us if you want to know more about AsyncAPI! The meeting is open to the public.

## Agenda

>Don't wait for the meeting to discuss topics that already have issues. Feel free to comment on them earlier.

1. Q&A
1. _(Leave a comment below to add your item to the agenda)_
1. Transfer of https://github.com/Pakisan/java-asyncapi from @Pakisan 
1. Plans for next 6 weeks aka Shape Up by @fmvilas 
1. Specification release schedule from @WaleedAshraf (related https://github.com/asyncapi/asyncapi-node/pull/24) 
1. Q&A

## Notes

- @Pakisan asked where official asyncapi logos can be found. @fmvilas provided link https://drive.google.com/drive/folders/17XAR-6D8Le7iQT3Y6laKNBsaGj_8DStF?usp=sharing
- we agreed with @Pakisan that the best is first to move the repo to AsyncAPI and work on improvements in AsyncAPI org. It will be better for Pavel as we will be able to work on asyncapi domain verification immediately and also there will be not much pressure in improvement and we will be able to adapt slowly forward.
- @fmvilas shared in details how we want to work on AsyncAPI forward as we grew from 2 to 4 people now. We just welcomed @jonaslagoni and @magicmatatjahu that in the past contributed to the project a lot. Watch the recording where Fran explains the Shape Up process that we choosed, and he explains https://shapeup.asyncapi.io and https://github.com/asyncapi/shape-up-process/issues. So far we just started so more details, explanation on the process will follow in January once we try it out and can actually share more on the topic.
- we talked about challanges with releasing new versions of specification, that we need to overcome and decide on before we can proceed with 2.1.0 release. @WaleedAshraf already created a followup issue

## Recording 

https://youtu.be/6STWIaqNPRc
